.eslintrc

```diff
@@ -64,7 +75,20 @@
     "react/wrap-multilines": 1,
     "semi": [2, "always"],
     "space-infix-ops": 0,
-    "quotes": [2, "single", "avoid-escape"]
+    "quotes": [2, "single", "avoid-escape"],
+    "eqeqeq": 1,
+    //jsdoc rules
+    "jsdoc/check-param-names": 1,
+    "jsdoc/check-tag-names": 1,
+    "jsdoc/check-types": 1,
+    "jsdoc/newline-after-description": 1,
+    "jsdoc/require-description-complete-sentence": 1,
+    "jsdoc/require-hyphen-before-param-description": 1,
+    "jsdoc/require-param": 2,
+    "jsdoc/require-param-description": 1,
+    "jsdoc/require-param-type": 1,
+    "jsdoc/require-returns-description": 1,
+    "jsdoc/require-returns-type": 1
```

Comment:
We might want to relax the rules a bit, otherwise we'll have too many warnings, and people will not pay attention to real problems.

I suggest the following defaults:
    "jsdoc/check-param-names": 2,
    "jsdoc/check-tag-names": 1,
    "jsdoc/check-types": 1,
    "jsdoc/newline-after-description": 0,
    "jsdoc/require-description-complete-sentence": 0,
    "jsdoc/require-hyphen-before-param-description": 0,
    "jsdoc/require-param": 2,
    "jsdoc/require-param-description": 0,
    "jsdoc/require-param-type": 0,
    "jsdoc/require-returns-description": 0,
    "jsdoc/require-returns-type": 1

We would still want to preserve those rules in the file, so that people who work on API documentation can turn them on to make docs consistent.

Reference link: https://github.com/gajus/eslint-plugin-jsdoc